Bender.yml

```diff
@@ -35,6 +35,7 @@ dependencies:
   redundancy_cells:       { git: "https://github.com/pulp-platform/redundancy_cells.git", rev: c37bdb47339bf70e8323de8df14ea8bbeafb6583 } # branch: astral_rebase
   redmule:                { git: "https://github.com/pulp-platform/redmule.git", rev: 7abb256 } # branch: fc/hci-v2
   neureka:                { git: "https://github.com/pulp-platform/neureka.git", rev: 60c51c0 } # branch: hci-v2
+  softex:                 { git: "https://github.com/belanoa/softex.git"       , rev: 9da2870 } # branch: master
```

Comment:
Before merging it's better to wait for this to be public (and possibly in the pulp namespace), do you agree @yvantor ?